src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaOfFigure, InfoFigure {
+    //круг
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double detAreaOfFigure() {
+        return Math.round(Math.PI * getRadius() * getRadius());
```

Comment:
The `detAreaOfFigure` method rounds the area to the nearest integer. If the task requires more precision, consider using a different rounding strategy or returning the exact value.